ftest.sh

```diff
@@ -56,8 +62,23 @@ cleanup() {
         fi
         ((i-=1)) || true
     done
+    if [ $i -eq 0 ]; then
+        echo "All cleanup attempts failed." >&2
+        return 1
+    fi
+    return 0
 }
 
+ls ~/.ssh
+touch ~/.ssh/known_hosts
+chmod 600 ~/.ssh/known_hosts
+for host in "$(IFS=','; echo "${nodes[*]}")"; do
```

Comment:
shellcheck is failing

Reply:
That code should have already been removed in one of the updates.   Apparently at some point in this PR a force push was done.

Reply:
If this code is still actually present, it needs to be removed.

Reply:
FIxed in 8a02226cb0c6403359067302488741deb3a7f52b
and removed in d929ddc18e80e16d84c3fb008137ed52a7242ad6